doc/reference/devicetree/api.rst

```diff
@@ -421,6 +421,9 @@ device.
      - Used by the OpenThread to specify UART device for Spinel protocol
    * - zephyr,pcie-controller
      - The node corresponding to the PCIe Controller
+   * - zephyr,rsc_table
```

Comment:
Is `zephyr` the correct vendor to use here? Resource table is more OpenAMP or Linux -defined than Zephyr

Reply:
Since the two names 'zephyr,ipc' and 'zephyr,ipc_shm' are already used for open-amp, I thought it is consistent to use them for the resource table as well.
But if I'm wrong and there's a better suggestion, I'll be happy to use it.